io-engine/src/subsys/nvmf/subsystem.rs

```diff
@@ -165,6 +196,186 @@ fn make_sn<T: AsRef<[u8]>>(uuid: T) -> CString {
 }
 
 impl NvmfSubsystem {
+    pub fn extract_nexus_name(nqn: &str) -> Option<String> {
```

Comment:
Might be worth moving this function somewhere else, perhaps close to where we define how our nqn is defined.

Reply:
moved to util::uri